enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/BaseHighLimitRecordFormat.java

```diff
@@ -182,7 +183,29 @@ public void write( RECORD record, PageCursor primaryCursor, int recordSize, Page
         }
         else
         {
-            markFirstByteAsUnused( primaryCursor );
+            markUnitsAsUnused( primaryCursor, storeFile, record, recordSize );
```

Comment:
What do you think about renaming `#markFirstByteAsUnused()` to `#markAsUnused()` and keep it as is in `BaseOneByteHeaderRecordFormat` but override in `BaseHighLimitRecordFormat` to also mark seconds unit as unused?

This will make subclasses less confused about which method to use `#markFirstByteAsUnused()` or `#markFirstByteAsUnused()`.